src/components/primitives/Text/Text.js

```diff
@@ -51,13 +60,10 @@ Text.propTypes = {
 	as: PropTypes.oneOfType([PropTypes.element, PropTypes.string]),
 	/** Variation */
 	variant: PropTypes.oneOf(['base', 'secondary', 'tertiary', 'error']),
-	align: PropTypes.oneOf(['left', 'center', 'right']),
 	children: PropTypes.node,
 };
 
-Text.defaultProps = {
-	variant: 'base',
-};
+Text.defaultProps = {};
```

Comment:
Any reason for removing default variant? It's quite convenient to use just `<Text>`.

Reply:
It is the same problem as before. Setting it to default here makes it impossible to override via props since it always takes precedence over them.

Reply:
Wait, it is a prop already, so it should be possible to override with a custom prop. We had problems with mixing props and styles. Or mixing default `m` with `mb` for example.

Reply:
when using `variant` it sets the styles as of variant and for some reason (order) those always take precendee over other props like `color` or `fontWeight`. At lest that was the reason to move those values to default part.

Reply:
Argh! So how can we have a default variant? What about using spread instead of `css`?

```
const common = {
  color: 'red'
}

variant({
  blue: {
      ...common,
     color: 'blue'
  }
})
```

Reply:
Why do we need a default variant in the first place? When you use a component without `variant` it's the default/

Reply:
Yeah, but it's kinda implicit and you kinda need two places to document it instead of saying: these are all available variants, this one is the default one.